addons/mail/static/src/core/common/messaging_parameters_service.js

```diff
@@ -0,0 +1,13 @@
+/* @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+export const messagingParametersService = {
```

Comment:
I don't like that we introduce this service again, call to `start` should just be moved to the correct bundle instead (both `public` and `web` but not common then)?

Reply:
I get your point but I'm not sure your suggestion is much better. We would need 1 patch per service that wants to start the bus per bundle. 

e.g

messaging in core does not start the bus
messaging in core/web does start the bus
messaging in discuss/core/public does start the bus.

Same holds for the "discuss_core_common service"..and so on. As for the handlers I don't think that is much of an issue to register them since nothing will be triggered before the start anyway and since it does not cost much to do so.  I applied your suggestion in order to show you the result.